posydon/interpolation/profile_interpolation.py

```diff
@@ -311,54 +352,57 @@ def __init__(self,initial,profiles,valid_initial,
             IF_interpolator (string) : path to .pkl file for IF interpolator for central density, final mass values
             n_comp (int) : number of PCA components. 
             hms_s2 (Boolean) : option to do profiles of star 2 in HMS-HMS grid
+            depth (int) : depth of neural network for principal component weights
+            width (int) : width of neural network for principal component weights
+            depthn (int) : depth of neural network for normalizing value
+            widthn (int) : width of neural network for normalizing value
+            lr (float) : learning rate for neural network training
```

Comment:
There are not mentioned variables: `mt`, `valid_mt` and a not required variable `lr`.

Reply:
good catch. just fixed this!